docs/examples/add-to-your-application/ruby/data_filtering.rb

```diff
@@ -0,0 +1,93 @@
+require 'oso'
+require 'sequel'
+require 'sinatra'
+
+require_relative './models'
+
+DB = Sequel.sqlite
+
+DB.create_table :repositories do
+  String :name
+  TrueClass :is_public
+end
+
+repositories = DB[:repositories]
+
+module DF
```

Comment:
Does this module need to exist? Is it just to avoid name conflicts with the `Repository` defined in `models.rb`?

Reply:
Yeah